plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -29,11 +29,11 @@ import (
 	gutil "github.com/gardener/gardener/pkg/utils/gardener"
 	. "github.com/gardener/gardener/pkg/utils/test/matchers"
 	. "github.com/gardener/gardener/plugin/pkg/shoot/validator"
-
```

Comment:
Can you please revert the changes in this file? They don't seem to be related to the PR.

Reply:
Sure. Initially there were changes to this file and later those were reverted back. I will revert all changes to this file.

Reply:
done